tutorials/nservicebus-101/index.md

```diff
@@ -4,32 +4,19 @@ suppressRelated: true
 reviewed: 2017-01-26
 ---
 
-In this course, you will learn how to get working quickly with NServiceBus.
+partial: nsb101-intro-paragraph
 
+ * In [Lesson 1: Getting started](lesson-1/), which should take no longer than 10-15 minutes, you will learn how to set up your development environment and create your very first messaging endpoint.
+ * In [Lesson 2: Sending a command](lesson-2/), which should take no longer than 15-20 minutes, you will learn how to define messages and message handlers. In the exercise, you'll send and process a `PlaceOrder` command within the same endpoint.
+ * In [Lesson 3: Multiple endpoints](lesson-3/), which should take no longer than 15-20 minutes, you will learn how to create multiple endpoints and send messages between them. In the exercise, you'll update your solution to send the `PlaceOrder` command to a separate **Sales** endpoint.
+ * In [Lesson 4: Publishing events](lesson-4/), which should take no longer than 25-30 minutes, you will learn how to publish events to multiple subscribers, and about the benefits of decoupling business processes using the Publish/Subscribe pattern. In the exercise, you'll modify your **Sales** endpoint to publish an `OrderPlaced` event to two new subscribers: a **Billing** endpoint for processing payments, and a **Shipping** endpoint to manage shipping items to customers.
+ * In [Lesson 5: Retrying errors](lesson-5/), which should take no longer than 25-30 minutes, you will learn how to deal with exceptions and take advantage of automatic retries. In the exercise, you'll throw throw an exception in a message handler and observe NServiceBus attempting to process the message several times before finally giving up on it and moving it aside to an error queue. Then you'll fix the exception and use the tools in the Particular Service Platform to replay the failed message through the original message handler successfully.
 
-## Objectives
+ When you've completed all the exercises, your solution will look like this:
 
-In this course we'll build a messaging system for a retail e-commerce domain.
-
-By the end of the course, you will have learned:
-
- * How to set up your development environment
- * What an endpoint is and how to create one
- * How to define messages and message handlers
- * How to send messages between endpoints
- * How (and why) to publish events to multiple subscribers
- * How to deal with exceptions and take advantage of automatic retries
-
-
-## Table of Contents
-
- * [Lesson 1: Getting started](lesson-1/)
- * [Lesson 2: Sending a command](lesson-2/)
- * [Lesson 3: Multiple endpoints](lesson-3/)
- * [Lesson 4: Publishing events](lesson-4/)
- * [Lesson 5: Retrying errors](lesson-5/)
+ ![Completed Solution Diagram](lesson-4/diagram.png)
 
 
 ## Get Started
 
-Go to [**Lesson 1: Getting started**](lesson-1/) to begin.
+If you'd like to get started right away, go to [**Lesson 1: Getting started**](lesson-1/) to begin.
```

Comment:
> If you'd like to get started right away

I prefer the original text. This is the bottom of the doc; there's nowhere else to go. Alternatively:

> Let's get started with Lesson 1

In which case, I'd consider dropping the heading too